modelzoo/template.py

```diff
@@ -0,0 +1,174 @@
+import argparse
+import tensorflow as tf
+import os
+
+# Set to INFO for tracking training, default is WARN. ERROR for least messages
+tf.logging.set_verbosity(tf.logging.INFO)
+print("Using TensorFlow version %s" % (tf.__version__))
+
+# Definition of some constants
+CONTINUOUS_COLUMNS = ["I" + str(i) for i in range(1, 14)]  # 1-13 inclusive
+CATEGORICAL_COLUMNS = ["C" + str(i) for i in range(1, 27)]  # 1-26 inclusive
+LABEL_COLUMN = ["clicked"]
+
+
+def add_layer_summary(value, tag):
+    tf.summary.scalar('%s/fraction_of_zero_values' % tag,
+                      tf.nn.zero_fraction(value))
+    tf.summary.histogram('%s/activation' % tag, value)
+
+
+# generate dataset pipline
+def generate_input_data():
+    pass
+
+
+# generate feature columns
+def build_feature_cols():
+    pass
+
+
+class WDL():
+    def __init__(self):
+        self.label = None
+        self.feature = None
+        self.bf16 = False
+        self._is_training = True
+
+        self.input_fn()
+        self.model_fn()
+        self.train_fn()
+        self.metrics_fn()
+
+    # process input data
+    def input_fn(self):
+        pass
+
+    # model structure
+    # finally get a probability of predict
```

Comment:
Is the comment correct? I think it aims to create a model.

Reply:
Yes, it does create a model. Which should be the output or the ending of the model? Logits(the input of the last sigmod), probability of prediction or prediction?